### ISSUE TITLE
Enter does not work while Basic access authentication pop up is open

### ISSUE THREAD

**Issue by lakospeter** [bitwarden/clients#12174]
### Steps To Reproduce

1. Open Chrome with an installed Bitwarden browser extension.
2. Log out from your Bitwarden browser extension if you're logged in.
3. Go to a site with Chrome browser that asks sign with **basic access authentication**.
4. Click on Bitwarden extension in Chrome's toolbar to open the Bitwarden pop-up.
5. Type your **Master password** correctly.
6. Push **Enter** button on your keyboard. (It just closes the Bitwarden pop-up but don't do the unlock, and you have to click on the **Unlock** link with your mouse in the Bitwarden pop-up to unlock your Bitwarden.)

### Expected Result

I should be able to do the unlock with pressing Enter after I typed my **Master password** correctly.

### Actual Result

It just close the Bitwarden pop-up but don't do the unlock.

### Screenshots or Videos

![20241128_103323](https://github.com/user-attachments/assets/155eede6-45cb-42a7-af52-e8c6f35c0ecb)

### Additional Context

Browser extension: Bitwarden Password Manager, Version 2024.11.2

### Operating System

Windows

### Operating System Version

Windows 10 Pro, 22H2, 19045.5131

### Web Browser

Chrome

### Browser Version

Version 131.0.6778.86

### Build Version

(Official Build) (64-bit)

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Krychaz** [comment]
Hi there,

I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by PegHorse** [comment]
![image](https://github.com/user-attachments/assets/9203b095-884f-4283-b48a-39b7008a133a)

Hi,
No detection of my elements in this Menu, while when i go to the Extension, i can see my ID and pass

![image](https://github.com/user-attachments/assets/9b03cb56-60bc-4438-b653-5a424b8a1690)

I have this bug in all websites since a while ago.
Bugged updates ?

My Browser is Brave.

**Comment by lakospeter** [comment]
@PegHorse Please, relocate your comment because it is out of context.

**Comment by PureLines** [comment]
> I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below.

@Krychaz I've had the same problem on Ubuntu 22.04, Chrome 134.0.6998.35, and the Bitwarden extension 2025.3.0 for several months (or longer).

**Comment by selckin** [comment]
Can confirm, having the basic auth popup open on *any* tab, prevents bitwarden from unlocking with pin/password, just closes the window and does nothing

**Comment by eddiert** [comment]
Still an issue and has been for a long time
Brave Browser 1.81.136 Chromium 139.0.7258.143 Bitwarden browser add-in 2025.8.1
Edge 1.39.0.3405.119 Bitwarden browser add-in 2025.8.1

Open https://authenticationtest.com/HTTPAuth/
Without closing username/password prompt click Bitwarden icon
Click master password field and type in master password
Press enter key and Bitwarden window closes silently
Bitwarden remains locked
Click Bitwarden icon again
Type in master password
Click Unlock button with mouse
Vault unlocks properly

Workaround
Click bitwarden icon
Type master password
Press tab-tab-space
Vault unlocks properly

Another symptom I just noticed. With basic auth prompt open Clicking the bitwarden icon doesn't automatically place the cursor into the master password field, you must manually click master password with the mouse before you start to type.

**Comment by lakospeter** [comment]
> Still an issue and has been for a long time Brave Browser 1.81.136 Chromium 139.0.7258.143 Bitwarden browser add-in 2025.8.1 Edge 1.39.0.3405.119 Bitwarden browser add-in 2025.8.1
> 
> Open https://authenticationtest.com/HTTPAuth/ Without closing username/password prompt click Bitwarden icon Click master password field and type in master password Press enter key and Bitwarden window closes silently Bitwarden remains locked Click Bitwarden icon again Type in master password Click Unlock button with mouse Vault unlocks properly
> 
> Workaround Click bitwarden icon Type master password Press tab-tab-space Vault unlocks properly

Another inconvenient workaround is to click the "Unlock" button with your mouse.

**Comment by michie-pk** [comment]
Browser: Chrome 141.0.7390.77 (Official Build) (64-bit)
Using Bitwarden browser extension, Version: 2025.9.0
SDK: 'main (ab3c7db)'
Server version: 2025.9.2

I can confirm the issue, with a slightly altered entrance to it.
I am logged out of the browser extension and I want to log in to have bitwarden available (in other words: I log in unprompted, by actively clicking on the browser extension icon).

- Hitting enter after entering the password closes the pop-up, but I am not logged in
- Tabbing to the "Login" Button and hitting enter also closes the pop-up, but I am not logged in
- Tabbing to the "Login" Button and hitting Space works - the pop-up stays open, and I am successfully logged in.

**Comment by scottwmaxwell** [comment]
Yes I more recently started experiencing this. I know it used to work, because I have muscle memory to just press the enter key.

**Comment by RIE-UPPERSOLUTIONS** [comment]
Any update on this issue? It's been plaguing me for over a year now.
Using Chrome on Windows.

**Comment by kielnino** [comment]
I can confirm the behavior described by @michie-pk. I have the same issue in Chrome on Windows.

**Comment by Haxthausen** [comment]
I've been experiencing the same issue with Chrome extension on Windows for long time.
Basically, as others explained, while vault is locked, simply pressing Enter on the extension pop-up after entering master password or pin closes the popup and does not unlock, and we have to manually press the Unlock button to successfully unlock.
It's quite annoying when you try to do most things via keyboard shortcuts.

Explicitly changing the focus to the Unlock button with Tab+Tab after entering master password or pin and pressing Enter also does not work.
However, I have noticed that doing the same and pressing Space instead works and successfully unlocks the vault, so can be a workaround.

**Comment by pamperer562580892423** [comment]
> ### Steps To Reproduce
> 1. Open Chrome with an installed Bitwarden browser extension.
> 2. Log out from your Bitwarden browser extension if you're logged in.
> 3. Go to a site with Chrome browser that asks sign with **basic access authentication**.
> 4. Click on Bitwarden extension in Chrome's toolbar to open the Bitwarden pop-up.
> 5. Type your **Master password** correctly.
> 6. Push **Enter** button on your keyboard. (It just closes the Bitwarden pop-up but don't do the unlock, and you have to click on the **Unlock** link with your mouse in the Bitwarden pop-up to unlock your Bitwarden.)

@lakospeter Honestly, I'm confused by your original "steps to reproduce". Because when you're **logged out** on the extension (step 2) you **can't _unlock_** your vault (steps 4-6), but only **log in**. [(unlock and log in are different things)](https://bitwarden.com/help/understand-log-in-vs-unlock/)

Could you clarify if you meant a **_locked vault_** in step 2?

**Comment by pamperer562580892423** [comment]
> Open https://authenticationtest.com/HTTPAuth/
> Without closing username/password prompt click Bitwarden icon
> Click master password field and type in master password
> Press enter key and Bitwarden window closes silently
> Bitwarden remains locked
> Click Bitwarden icon again
> Type in master password
> Click Unlock button with mouse
> Vault unlocks properly

I just tried to reproduce what @eddiert wrote here, and could successfully unlock my locked vault with the master password and just hitting the return or enter key.

BW browser extension Version: 2026.7.0, SDK: 'main (3e060f8)', Server version: 2026.7.2
Vivaldi 8.1.4087.62 (Official Build) (64-bit)
Windows 11 Home 25H2

**Comment by lakospeter** [comment]
> > ### Steps To Reproduce
> > 
> > 1. Open Chrome with an installed Bitwarden browser extension.
> > 2. Log out from your Bitwarden browser extension if you're logged in.
> > 3. Go to a site with Chrome browser that asks sign with **basic access authentication**.
> > 4. Click on Bitwarden extension in Chrome's toolbar to open the Bitwarden pop-up.
> > 5. Type your **Master password** correctly.
> > 6. Push **Enter** button on your keyboard. (It just closes the Bitwarden pop-up but don't do the unlock, and you have to click on the **Unlock** link with your mouse in the Bitwarden pop-up to unlock your Bitwarden.)
> 
> [@lakospeter](https://github.com/lakospeter) Honestly, I'm confused by your original "steps to reproduce". Because when you're **logged out** on the extension (step 2) you **can't _unlock_** your vault steps 4-6, but only **log in**. [(unlock and log in are different things)](https://bitwarden.com/help/understand-log-in-vs-unlock/)
> 
> Could you clarify if you meant a **_locked vault_** in step 2?

You're right. I correct the text of **Step 2** for the following: "**Lock** your Bitwarden in the browser extension if you're in unlocked state."
You have to see the following screen at **Step 4**:
<img width="1920" height="1032" alt="Image" src="https://github.com/user-attachments/assets/e3dd43d4-a7d2-4c63-87d0-2844708bfdd5" />
